wordpress-mobile/WordPress-Android
# Issue/13328 restore complete

Parent #13328

This PR implements the "complete" wizard step/view for the restore request and is mostly a clone of Backup Download complete. This PR also undoes the temporary navigations actions implemented in the progress PR #13778 .

**Merge Instructions**
- Make sure #13773, #13775 & #13778 have been merged (sorry, but they need to build upon each other)
- Remove Not Ready For Merge Label
- Merge as normal

**Cloned Classes**
RestoreCompleteAdapter
RestoreCompleteFragment
RestoreCompleteStateListItemBuilder
RestoreCompleteViewModel

**Notes**:
- Ignore styling - this is being handled separately 
- Ignore unit test placement in packages - this is being handled separately 
- RestoreActivity & RestoreViewModel (+unit tests) will continue to get updated as I move throughout the steps of the wizard

**To test:**
**Warning**: The Restore site button is hooked up, so if you press it your site will be backed up!
- Launch the app with RestoreFeatureConfig flag on and the BackupFeatureFlag on
- Navigate to ActivityLog
- Tap the more menu
- Select the restore option
- Tap the Restore Site button on the Details View
- Tap the Confirm restore site on the Warning View
- The Progress view will do it's thing - don't cancel out
Verify
- (1) The complete view is shown when the restore is finished
- (2) Tapping the Visit Site button to launch the site url in an external browser
- (3) Tap the Done button to close the wizard

PR submission checklist:
